Remotion/ObjectBinding/Web/UI/Controls/BocRendererBase.cs

```diff
@@ -158,7 +158,16 @@ public virtual string CssClassReadOnly
     /// </remarks>
     public virtual string CssClassDisabled
     {
-      get { return "disabled"; }
+      get { return CssClassDefinition.Disabled; }
+    }
+
+    /// <summary>
+    /// Gets whether the control establishes a remotion theming context which automatically styles standard browser controls (e.g. input elements)
+    /// in the current theme. <c>false</c> by default. Derived classes may override this behavior. 
+    /// </summary>
+    public virtual bool EstablishesRemotionThemingContext
```

Comment:
protected, please. 

Also, I can see why this was introduced this way. It feels a bit weird, but it's probably the best way to allow the controls to manipulate the behavior of `OverrideCssClass`.